### ISSUE TITLE
brands.agents.getVerification not returning verificationstate

### ISSUE THREAD

**Issue by ghost** [google-business-communications/nodejs-businesscommunications#1]
brands.agents.getVerification  is not returning verificationstate it is returning agent object.
same with the case of brands.locations.getVerification DOESNOT return verificationState , it is also returning location object

**Comment by thefalc** [comment]
Hey there, thanks for the feedback. I took a look at this and it appears to be working for me. Below is the code I used for the getVerification request for an agent. Make sure you are appending '/verification' to the name field as shown in the spec: https://developers.google.com/business-communications/business-messages/reference/business-communications/rest/v1/brands.agents/getVerification

const apiConnector = apiHelper.init();
apiConnector.then((apiObject) => {
    const apiParams = {
      auth: apiObject.authClient,
      name: agentId + '/verification'
    };
 
    apiObject.bcApi.brands.agents.getVerification(apiParams, {}, (err, response) => {
        const agentVerification = response.data;

        console.dir(agentVerification);
      });
});
